File: Gestion Cours-WEB/src/Entity/SalleDeSport.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * SalleDeSport
 *
 * @ORM\Table(name="salledesport")
 * @ORM\Entity
 */
class SalleDeSport
{
    /**
     * @var int
     *
     * @ORM\Column(name="ID_Salle", type="integer", nullable=false)
     */
    private $idSalle;

    /**
     * @var string
     *
     * @ORM\Column(name="Nom_Salle", type="string", length=50, nullable=false)
     * @ORM\Id
     */
    private $nomSalle;

    /**
     * @var string
     *
     * @ORM\Column(name="Adresse", type="string", length=50, nullable=false)
     */
    private $adresse;

    /**
     * @var string
     *
     * @ORM\Column(name="HDebut", type="string", length=50, nullable=false)
     */
    private $hdebut;

    /**
     * @var string
     *
     * @ORM\Column(name="HFin", type="string", length=50, nullable=false)
     */
    private $hfin;

    /**
     * @var string
     *
     * @ORM\Column(name="Minn", type="string", length=50, nullable=false)
     */
    private $min;

    /**
     * @var string
     *
     * @ORM\Column(name="NumTel", type="string", length=50, nullable=false)
     */
    private $numtel;

    /**
     * @return int
     */
    public function getIdSalle(): ?int
    {
        return $this->idSalle;
    }

    /**
     * @param int $idSalle
     */
    public function setIdSalle(?int $idSalle): void
    {
        $this->idSalle = $idSalle;
    }

    /**
     * @return string
     */
    public function getNomSalle(): ?string
    {
        return $this->nomSalle;
    }

    /**
     * @param string $nomSalle
     */
    public function setNomSalle(?string $nomSalle): void
    {
        $this->nomSalle = $nomSalle;
    }

    /**
     * @return string
     */
    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    /**
     * @param string $adresse
     */
    public function setAdresse(?string $adresse): void
    {
        $this->adresse = $adresse;
    }

    /**
     * @return string
     */
    public function getHdebut(): ?string
    {
        return $this->hdebut;
    }

    /**
     * @param string $hdebut
     */
    public function setHdebut(?string $hdebut): void
    {
        $this->hdebut = $hdebut;
    }

    /**
     * @return string
     */
    public function getHfin(): ?string
    {
        return $this->hfin;
    }

    /**
     * @param string $hfin
     */
    public function setHfin(?string $hfin): void
    {
        $this->hfin = $hfin;
    }

    /**
     * @return string
     */
    public function getMin(): ?string
    {
        return $this->min;
    }

    /**
     * @param string $min
     */
    public function setMin(?string $min): void
    {
        $this->min = $min;
    }

    /**
     * @return string
     */
    public function getNumtel(): ?string
    {
        return $this->numtel;
    }

    /**
     * @param string $numtel
     */
    public function setNumtel(?string $numtel): void
    {
        $this->numtel = $numtel;
    }

    public function __toString() {
        return $this->getNomSalle();
    }


}

File: mobilefinaleintegration/src/com/mycompany/myapp/entities/Salledesport.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author asus
 */
public class Salledesport {
    private String idSalle;
    private String nomSalle;
    private String adresse;
    private String region;
    private int hdebut;
    private int hfin;
    private int min;
    private int numtel;

    public Salledesport(String idSalle, String nomSalle, String adresse, String region, int hdebut, int hfin, int min, int numtel) {
        this.idSalle = idSalle;
        this.nomSalle = nomSalle;
        this.adresse = adresse;
        this.region = region;
        this.hdebut = hdebut;
        this.hfin = hfin;
        this.min = min;
        this.numtel = numtel;
    }

    public Salledesport() {
    }

    public String getIdSalle() {
        return idSalle;
    }

    public void setIdSalle(String idSalle) {
        this.idSalle = idSalle;
    }

    public String getNomSalle() {
        return nomSalle;
    }

    public void setNomSalle(String nomSalle) {
        this.nomSalle = nomSalle;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getRegion() {
        return region;
    }

    public void setRegion(String region) {
        this.region = region;
    }

    public int getHdebut() {
        return hdebut;
    }

    public void setHdebut(int hdebut) {
        this.hdebut = hdebut;
    }

    public int getHfin() {
        return hfin;
    }

    public void setHfin(int hfin) {
        this.hfin = hfin;
    }

    public int getMin() {
        return min;
    }

    public void setMin(int min) {
        this.min = min;
    }

    public int getNumtel() {
        return numtel;
    }

    public void setNumtel(int numtel) {
        this.numtel = numtel;
    }
    
      @Override
    public String toString() {
        return "Salledesport{" + "IdSalle=" + idSalle + ", NomSalle=" + nomSalle + ", Adresse=" + adresse + ", Région=" + region + ", Heure Début=" + hdebut + ", Heure Fin=" + hfin + ", Min=" + min + ", N°Tel=" + numtel + '}';
    }
}
